src/data/DBHelpers.hpp

```diff
@@ -107,6 +107,7 @@ struct NFTsData {
     ripple::AccountID owner;
     std::optional<ripple::Blob> uri;
     bool isBurned = false;
+    bool onlyUriChanged = false;  // Whether only the URI was changed
```

Comment:
After we move to ETL plugins, this intermediate data structure will be replaced by the interface like `onNFTModifyTransaction` `onNFTMintTransaction`.